src/reviewerQueue.js

```diff
@@ -16,27 +16,25 @@
 //   pcsforeducation
 
 const _ = require('lodash');
-const GitHubApi = require('github');
-const {fetchTravelEvents} = require('./lib/googleCalendar');
+const {Octokit} = require('@octokit/rest');
+const shadows = require('./shadows');
```

Comment:
Hmm so it looks like all Hubot config is through env variables. Maybe we can make this either a stringified version of the object in shadows or a path to a JSON file to load.

Reply:
Hmmm, so the shadows resource would be another environment variable? Interesting. I know there was an email map as an environment variable before for the Gcal/vacation stuff - could probably work similarly to how that was done?

Reply:
Done. Shadows is now contained in an environment variable.